types/BountyGoal.d.ts

```diff
@@ -0,0 +1,17 @@
+import { IQuestGoalConfig, IQuestGoalState, QuestGoal } from '../src/QuestGoal';
+
+export interface IBountyGoalConfig extends IQuestGoalConfig {
```

Comment:
Yes, those come with the bundle-example-quests library. We'll have to figure out one day how to handle everything since it's not a mono repo. But seemed like a waste for me to make those types on my private repo and not include them for others.